app/services/get_embedded_content_service.rb

```diff
@@ -0,0 +1,27 @@
+class GetEmbeddedContentService
+  attr_reader :target_content_id
+
+  def initialize(target_content_id)
+    self.target_content_id = target_content_id
+  end
+
+  def call
+    if Document.find_by(content_id: target_content_id).nil?
+      message = "Could not find an edition to get embedded content for"
+      raise CommandError.new(code: 404, message:)
+    end
+
+    Presenters::EmbeddedContentPresenter.present(
+      target_content_id,
+      host_editions,
```

Comment:
just a minor suggestion but I guess this isn't technically `editions` , but a specific type of Data object that has both Document and Edition fields, maybe should say `host_content`?